### ISSUE TITLE
cci org list should show a friendlier error if run in a project not configured for CumulusCI

### ISSUE THREAD

**Issue by jlantz** [SFDO-Tooling/CumulusCI#709]
I think after the recent changes to initializing the project config, we're no longer catching and handling errors in cci org list if the project isn't configured.

```
$ cci org list
Traceback (most recent call last):
  File "/Users/jlantz/.pyenv/versions/cci/bin/cci", line 11, in <module>
    sys.exit(main())
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/Users/jlantz/.pyenv/versions/2.7.14/envs/cci/lib/python2.7/site-packages/cumulusci/cli/cci.py", line 687, in org_list
    for org in config.project_config.list_orgs():
AttributeError: 'NoneType' object has no attribute 'list_orgs'
```

This used to throw the following error:
```Error: No project configuration found.  You can use the "project init" command to initilize the project for use with CumulusCI```

**Comment by davisagli** [comment]
I think this is addressed by #788

**Comment by cdcarter** [comment]
(cci-q4) ➜  Products cci org list
Checking the version!
No repository found in current path.  You must be inside a repository to initialize the project configuration
